index.html

```diff
@@ -0,0 +1,227 @@
+            </div>
+            <div class="logo">
+                <span><img src="./assets/images/cvimage2.JPG" alt="logo"></span>
+            </div>
+            <ul class="nav_list">
+                <li class="navlink"> <a href="#">Portfolio</a></li>
+                <li class="navlink"> <a href="#">Contact</a></li>
+                <li class="navlink"> <a href="#">About</a></li>
+            </ul>
+            
+        </nav>
+    </header>
+    <!-- Headline_section -->
+    <section class="main_containers">
+        <div class="headline_container">
+            <h1 class="primary-text">
+                <span>I'm Bett Enock. <br/>
+                Glad to see you!
+                </span></h1>
+            <p class="secondary_text">
+                I’m a software developer! I can help you to build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hesitate to contact
+                me.
+              </p>
+              <div class="social_media">
+                <h2 class="tertiary-text">Let's connect</h2>
+                <ul class="socialmedia-icons">
+                    <li><a href="#"><img src="./assets/images/github.svg" alt="github"></a> </li>
+                    <li><a href="#"><img src="./assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                    <li><a href="#"><img src="./assets/images/angel list.svg" alt="angellist"></a> </li>
+                    <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
+                </ul>
+              </div>
+
+        </div>
+    </section>
+    <!-- Headline Section -->
+    <Section class="main_containers">
+        <div class="wrapper-grid">
+            <div class="gridcontainer">
+                <div class="snapshot">
+                    <img class="snapshot1" src="./assets/images/Snapshoot Portfolio.png" alt="snapshot">
+                </div>
+                <div class="grid-items">
+                    <h2>Tonic</h2>
+                    <ul class="snapshot-list content1">
+                        <li class="Company">CANOPY</li>
+                        <li class="description">Back End Dev</li>
+                        <li class="date">2015</li>
+                    </ul>
+                    <p class="snapshot-text">
+                        A daily selection of privately 
+                        personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="snapshot-list content2">
+                        <li class="Company">HTML</li>
+                        <li class="description">CSS</li>
+                        <li class="date">Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+                
+            </div>
```

Comment:
[ ] Please note that you are required to resemble the Figma design as much as possible.
- The contents of your page are not properly aligned as in the Figma design
- There is too much space between your content and the edges
- The topic on your application is not as bold as it is on the Figma design
- The font of your description text is different from the Figma design
- The list of technologies on your application is not the same color as the Figma design
- The `see project` button on your application is different from the Figma design
- The text underneath the topic does not resemble the Figma illustrations

###
> This is the requirement
![Screenshot from 2023-06-03 17-02-23](https://github.com/Bettenoch/My_Portfolio/assets/69245974/e7b016d9-6b5e-4ee3-ba10-85fae7c939e7)

###
> This is your design
![Screenshot from 2023-06-03 16-51-39](https://github.com/Bettenoch/My_Portfolio/assets/69245974/473f4475-4676-445d-b0bb-6eeafc4b9d92)

###
> This is the Figma design
![Screenshot from 2023-06-03 16-46-29](https://github.com/Bettenoch/My_Portfolio/assets/69245974/c33163de-c027-4db7-92b3-2ba01689a287)